parkings/tests/test_models.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+
+from django.contrib.gis.geos import Point
+from django.utils.timezone import now
+
+from parkings.models import Address, Operator, Parking
+
+
+class TestAddressModel(object):
```

Comment:
Sure, can do 👍 

Pytest actually does recommend [grouping tests in classes](http://doc.pytest.org/en/latest/getting-started.html#grouping-multiple-tests-in-a-class) for maintainability. It helps especially when the project becomes more complex and the amount of tests is higher. But functions are ok too, changed 👍